From 86a017ab7b591c00f4802c65392671636cd2c5fe Mon Sep 17 00:00:00 2001
From: Wanwei Jiang <wanwei.jiang@amlogic.com>
Date: Tue, 7 Feb 2023 17:19:36 +0800
Subject: [PATCH] Merge branch 'android14-5.15' into amlogic-5.15-dev [1/1]

PD#SWPL-87897

Problem:
record merge upstream information

Solution:
AML_KERNEL_VERSION      12
UPSTREAM_VERSION        "2447b4c0a5ca8"
AML_PATCH_VERSION       "2447b4c0a5ca8"
MERGE_DATE              "2024-08-02"

Verify:
sc2

Change-Id: I69f8a7d5d122dec79a63283b9def5eea59a50559
Signed-off-by: Wanwei Jiang <wanwei.jiang@amlogic.com>
---
 include/linux/upstream_version.h | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 include/linux/upstream_version.h

diff --git a/include/linux/upstream_version.h b/include/linux/upstream_version.h
new file mode 100644
index 0000000000000..6f54eab0f2adc
--- /dev/null
+++ b/include/linux/upstream_version.h
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: (GPL-2.0+ OR MIT) */
+/*
+ * Copyright (c) 2019 Amlogic, Inc. All rights reserved.
+ */
+
+#ifndef _LINUX_UPSTREAM_VERSION_H
+#define _LINUX_UPSTREAM_VERSION_H
+
+#define AML_KERNEL_VERSION	12
+#define UPSTREAM_VERSION	"2447b4c0a5ca8"
+#define AML_PATCH_VERSION	"2447b4c0a5ca8"
+#define MERGE_DATE		"2024-08-02"
+
+#endif
-- 
2.42.0

